FoodBookApp/UI/Views/ForYouView.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ForYouView.swift
+//  FoodBookApp
+//
+//  Created by Juan Diego Yepes Parra on 21/02/24.
+//
+
+import SwiftUI
+
+struct ForYouView: View {
+    var body: some View {
+        ZStack {
+            
+            Image(systemName: "star")
+                .foregroundColor(/*@START_MENU_TOKEN@*/.blue/*@END_MENU_TOKEN@*/)
```

Comment:
im not clear on what @START_MENU_TOKEN does, please add an inline comment explaining (ideally everywhere its used)

Reply:
Besti i do NOT know that that is, i guess it looks like that in source code but in swift this is the syntax:

<img width="378" alt="Screenshot 2024-02-22 at 7 28 20 AM" src="https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/77981728/843e5e4b-2c99-45f7-ae1c-f31206da4980">

Reply:
let me clone real quick and check to see what i get

Reply:
yes on local its normal, should be some underlying syntax